index.html

```diff
@@ -168,7 +168,29 @@ <h4>LET'S CONNECT</h4>
         <li><img src="./images/Frame12.svg" alt=""></li>
       </ul>
     </div>
-  </div> 
-</section>   
+  </div>
+   <div class="contact-main">
+    <div class="contact-me">
+      <div class="text-contact">
+        <img id="contact-img" src="./images/PrimaryText-1.svg" alt="Contact-me">
+        <br>
+        <img src="./images/PrimaryText-21.svg" alt="text">
+      </div>
+      <div class="form">
+        <form action="https://formspree.io/f/mknyrjvn" method="post">
+        <ul>
+          <li><input type="text" maxlength="30" placeholder="Name" required></li>
+          <li><input type="email" placeholder="Enter E-mail" required></li>
+          <li><textarea name="msg" placeholder="Write your message here" maxlength="500"></textarea></li>
```

Comment:
- [ ] Kindly make this input field `required` so that the form can't be submitted when this field is empty.